doc/wifi/wifi.doxyfile.in

```diff
@@ -1020,7 +1020,7 @@ RECURSIVE              = YES
 # Note that relative paths are relative to the directory from which Doxygen is
 # run.
 
-EXCLUDE                =
+EXCLUDE                = bus_if/bus/pcie/inc/pcie.h
```

Comment:
@kapbh the path should be relative to the directory where doxygen is run, maybe use `@DOCSET_SOURCE_BASE@/bus_if/bus/pcie/inc/pcie.h` ?